srcs/initializers.c

```diff
@@ -33,6 +33,14 @@ t_cub	init_cub(void)
 	cub.side_dist = (t_pair_d *)calloc_or_exit(sizeof(t_pair_d), 1, &cub);
 	cub.delta_dist = (t_pair_d *)calloc_or_exit(sizeof(t_pair_d), 1, &cub);
 	cub.step = (t_pair_i *)calloc_or_exit(sizeof(t_pair_i), 1, &cub);
+	cub.keys.w = 0;
```

Comment:
let's make a init_keys function which is called here, else we go over the 25 lines 😅

Reply:
Done!